src/Tmds.Ssh/PrivateKeyCredential.cs

```diff
@@ -2,15 +2,21 @@
 // See file LICENSE for full license details.
 
 using System;
+using System.Text;
 
 namespace Tmds.Ssh;
 
-public sealed class PrivateKeyCredential : Credential
+public sealed partial class PrivateKeyCredential : Credential
 {
     internal string FilePath { get; }
+    internal byte[] Passphrase { get; }
 
-    public PrivateKeyCredential(string path)
+    public PrivateKeyCredential(string path) : this(path, null)
+    { }
+
+    public PrivateKeyCredential(string path, string? passphrase)
```

Comment:
Can we use 'password', like `X509Certificate2` ctor?

Reply:
I can change it if you like. The reason why I went with `passphrase` is that's the wording OpenSSH uses for encrypted private keys.

Reply:
I prefer `password` because that is what .NET BCL API use.